### DISCUSSION TITLE
Screen Tearing even after TearFree enabled

### DISCUSSION THREAD

**Discussion by decoded-satapathy** [i3/i3#5662] [Q&A]
I recently moved from Gnome to i3wm. And as soon as I started using it, I observed screen tearing. I am using Intel's inbuilt Iris Xe Graphics with picom as my compositor ( with vsync as true in it's .conf file). After some research, I found that it is due to Xorg as it supports screen tearing in order to prevent latency. I also found that you can change it by going into your 
`/etc/X11/xorg.conf.d` and in that create a file named `20-intel.conf` with the following contents

```
Section "Device"
    Identifier    "Intel Graphics"
    Driver        "intel"
    Option        "TearFree"  "true"
EndSection
```

but even after this and then rebooting, I am experiencing screen tearing (not severe but still enough to ruin the experience)

I also saw in a video that this could be because my display is 2240 x 1400 with 60 Hz. Either decreasing the resolution (which I don't want) or increasing the refresh hertz (which I can not do ) can fix the problem.

I saw another video where I learned that an intel driver package (`xserver-xorg-video-intel`) could cause some issues and suggested removing it, I tried it but it made me unable to log back into my system and I reinstalled it.

Anyone who can help me, kindly show me what to do. Thanks in advance.

**Comment by orestisfl** [comment]
What backend are you using with picom? Sharing your config might help.

I would give a read to this section of the arch linux article for intel graphics: https://wiki.archlinux.org/title/intel_graphics#Tearing. AFAIK, the alternative of using the modesetting `TearFree` optionis not released yet.

**Comment by decoded-satapathy** [comment]
I tried to use `Driver "modsetting"` instead of `Driver "intel"` ,but it didn't make any difference (in my opinion it made it worse). I was sick of the screen tearing and upon more research I found that it is due to the way X11 display server runs. Hence I switched to wayland using Sway (good bye to i3)
@orestisfl  Thanks for the help man, your reply opened my eyes to the wonderful world of arch wiki (earlier thought it was only helpful for arch and arch based distros)
